### PR TITLE
Enable ci-operator image-registry job

### PR DESCRIPTION
**Pull request by smarterclayton**
Hold for soaking

### CODE REVIEW

**Comment by smarterclayton** [comment]
Also reorder the containers so the test container is first.

**Comment by kargakis** [comment]
How is the ci operator using cluster/ci/config/prow/jobs/cluster-launch-e2e.yaml?

**Comment by kargakis** [comment]
> How is the ci operator using cluster/ci/config/prow/jobs/cluster-launch-e2e.yaml?

To answer my own question, cluster/ci/config/prow/jobs/cluster-launch-e2e.yaml is mounted in the ci-operator via a config map and then presumably the ci-operator knows how to process and apply the template. What moves the template inside the configmap?

**Comment by kargakis** [comment]
> What moves the template inside the configmap?

https://github.com/openshift/release/blob/42503ba0787f2a4b7c5a14c1837c4a75abd580a6/Makefile#L140

File: Makefile [hunk 1]
```diff
@@ -147,9 +147,13 @@ prow-jobs: prow-cluster-jobs
 	$(MAKE) applyTemplate WHAT=projects/acs-engine/build.yaml
 .PHONY: prow-jobs
 
-projects: gcsweb kube-state-metrics oauth-proxy origin-stable origin-release prometheus test-bases image-pruner-setup autoscaler descheduler node-problem-detector publishing-bot cluster-capacity content-mirror
+projects: gcsweb kube-state-metrics oauth-proxy origin-stable origin-release prometheus test-bases image-pruner-setup autoscaler descheduler node-problem-detector publishing-bot cluster-capacity content-mirror image-registry
 .PHONY: projects
 
+image-registry:
```
[the remark this answers is not in the document]
Comment (smarterclayton) [reply Makefile]: My proposal here: https://github.com/openshift/ci-operator/issues/17

**Pull request merged**